docs/source/examples/checkpointer.rst

```diff
@@ -0,0 +1,319 @@
+.. _understand_checkpointer:
+
+==============================
+Understanding the Checkpointer
+==============================
+
+This tutorial will walk you through the design and behavior of the checkpointer and associated
+utilities.
+
+
+Overview
+--------
+
+TorchTune checkpointers are designed to be composable components which can be plugged
+into any recipe - training, evaluation or generation. Each checkpointer supports a
+set of models and scenarios making these easy to understand, debug and extend.
+
+TorchTune is designed to be "state-dict invariant".
+
+- At the input, TorchTune accepts checkpoints from multiple sources in multiple formats.
+  For Llama2 this includes both the HF Hub and the Meta Llama website. Model users don't
```

Comment:
nit: prefer to write to the reader (i.e. "Model users don't have to worry about" -> "You don't have to worry about")